pkg/config.go

```diff
@@ -39,6 +39,35 @@ func (*Config) Project() string {
 	return os.Getenv("GOOGLE_PROJECT")
 }
 
+// TlsVersion will be used current TLS Policy
+// by this value. The acceptable value MUST BE TLS 1.1/1.2/1.3
+// Note that the TLS 1.3 supports `TLS_AES_128_GCM_SHA256`,
+// `TLS_AES_256_GCM_SHA384` and `TLS_CHACHA20_POLY1305_SHA256` ciphers only.
+// Default is "TLS_1_2"
+func (*Config) TlsVersion() string {
```

Comment:
nitpick: best practice for comment looks something like this

```
// Fprint formats using the default formats for its operands and writes to w.
// Spaces are added between operands when neither is a string.
// It returns the number of bytes written and any write error encountered.
func Fprint(w io.Writer, a ...interface{}) (n int, err error) {
```

Reply:
Fixed by https://github.com/commercetools/gcp-ssl-policy-asserter/pull/3/commits/6f590c87e8855b8392419a2bb64a92a384f2a23c